openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/TeamResource.java

```diff
@@ -666,6 +668,69 @@ public CsvImportResult importCsv(
     return importCsvInternal(securityContext, name, csv, dryRun);
   }
 
+  @PUT
```

Comment:
@sonika-shah lets add a GET and remove users from fields for TeamResource, UI should make the call to get the user list with pagination

Reply:
<img width="727" alt="image" src="https://github.com/user-attachments/assets/03604e19-a4eb-4707-8a4d-71aabb01f921">
yes @harshach , will have a separate api to GET the users list for a particular team with pagination, will also need the above PUT to update users in the team  (to avoid using the patch call to update users in a team)